### ISSUE TITLE
[Question]: How Ernie-layout create FUNSD training and evaluation split?

### ISSUE THREAD

**Issue by htcml** [PaddlePaddle/PaddleNLP#5813]
### 请提出你的问题

When I run [the script](https://github.com/PaddlePaddle/PaddleNLP/tree/develop/model_zoo/ernie-layout#41-key-information-extraction) to create fine-tuned Ernie-layout model on FUNSD. The following output shows that it has 152 examples for training and 59 examples for evaluation. But FUNSD has 149 in training folder and 50 in testing folder. So just wonder how do you create 152/59 split? Do they overlap?

```
[2023-04-28 04:34:17,137] [    INFO] - The following columns in the training set  don't have a corresponding argument in `ErnieLayoutForTokenClassification.forward` and have been ignored: token_is_max_context, token_to_orig_map, id, tokens. If token_is_max_context, token_to_orig_map, id, tokens are not expected by `ErnieLayoutForTokenClassification.forward`,  you can safely ignore this message.
[2023-04-28 04:34:17,198] [    INFO] - ***** Running training *****
[2023-04-28 04:34:17,198] [    INFO] -   Num examples = 152
[2023-04-28 04:34:17,198] [    INFO] -   Num Epochs = 264
[2023-04-28 04:34:17,198] [    INFO] -   Instantaneous batch size per device = 4
[2023-04-28 04:34:17,198] [    INFO] -   Total train batch size (w. parallel, distributed & accumulation) = 4
[2023-04-28 04:34:17,199] [    INFO] -   Gradient Accumulation steps = 1
[2023-04-28 04:34:17,199] [    INFO] -   Total optimization steps = 10000
[2023-04-28 04:34:17,199] [    INFO] -   Total num train samples = 40000
[2023-04-28 04:34:17,209] [    INFO] -   Number of trainable parameters = 281696967
  1% 100/10000 [01:31<2:20:16,  1.18it/s][2023-04-28 04:35:49,178] [    INFO] - The following columns in the evaluation set  don't have a corresponding argument in `ErnieLayoutForTokenClassification.forward` and have been ignored: token_is_max_context, token_to_orig_map, id, tokens. If token_is_max_context, token_to_orig_map, id, tokens are not expected by `ErnieLayoutForTokenClassification.forward`,  you can safely ignore this message.
[2023-04-28 04:35:49,333] [    INFO] - ***** Running Evaluation *****
[2023-04-28 04:35:49,333] [    INFO] -   Num examples = 59
[2023-04-28 04:35:49,333] [    INFO] -   Total prediction steps = 15
[2023-04-28 04:35:49,333] [    INFO] -   Pre device batch size = 4
[2023-04-28 04:35:49,333] [    INFO] -   Total Batch size = 4
```

**Comment by linjieccc** [comment]
@htcml Hi, the original training set and validation set consisted of 149 and 50 samples respectively. During the data preprocessing stage, we incorporated [sliding window processing](https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/ernie-layout/utils.py#L319) to segment some of the texts with length exceeding max_length. Therefore, the number of training samples in the fine-tuning stage will be greater than or equal to the original sample size.
